tensorrt_llm/_torch/modules/fused_moe/fused_moe_cute_dsl.py

```diff
@@ -0,0 +1,271 @@
+import math
+from typing import List, Optional, Union
+
+import torch
+import torch.nn.functional as F
+
+from ...distributed import allgather
+from ...model_config import ModelConfig
+from ...utils import Fp4QuantizedTensor, disable_fp4_allgather, reswizzle_sf
+from .fused_moe_cutlass import CutlassFusedMoE
+from .quantization import MoEWeightLoadingMode
+from .routing import BaseMoeRoutingMethod
+
+
+def swiglu_fused_moe(x):
+    x, gate = x.chunk(2, dim=-1)
+    return F.silu(gate) * x
+
+
+def cute_dsl_fp8_group_blockwise_gemm_ref(
```

Comment:
NIT. This should go to tests directory right?

Reply:
It is needed for the overall flow now, especially for model accuracy test now. In the end, it will replaced with optimized cute_dsl kernels.